packages/lambda-xyz/src/__test__/xyz.test.ts

```diff
@@ -48,15 +36,15 @@ o.spec('LambdaXyz', () => {
         for (const tileSetName of TileSetNames) {
             const tileSet = new TileSet(tileSetName, EPSG.Google, 'bucket');
             TileSets.set(tileSet.id, tileSet);
-            tileSet.load = () => Promise.resolve();
+            tileSet.load = () => Promise.resolve(true);
             tileSet.getTiffsForQuadKey = async (): Promise<[]> => [];
         }
     });
 
     o.afterEach(() => {
         TileSets.clear();
-        Tilers.tile256 = new Tiler(256);
-        Tilers.compose256 = new TileMakerSharp(256);
+        Tilers.tile256 = origTile256;
```

Comment:
These shouldn't have internal state, so should just be able to recreate them.

Reply:
Yeah but then if you change them to something else you have to remember to change it here too. This way we are confident that it is always restored to what it was.